### PR TITLE
Resize live volumes attached to an instance

### PR DESCRIPTION
**Pull request by rnirmal**

### CODE REVIEW

File: reddwarf/db/sqlalchemy/session.py [hunk 1]
```diff
@@ -82,10 +82,12 @@ def _create_engine(options):
...
 def get_session(autocommit=True, expire_on_commit=False):
     """Helper method to grab session."""
-
     global _MAKER, _ENGINE
     if not _MAKER:
-        assert _ENGINE
+        if not _ENGINE:
+            msg = "***The Database has not been setup!!!***"
+            LOG.exception(msg)
+            raise RuntimeError(msg)
```
Comment (TimSimpsonR) [inline reddwarf/db/sqlalchemy/session.py:90]: A real error message! Kudos.

**Comment by TimSimpsonR** [comment]
+1

**Comment by sacharya** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Comment by cp16net** [comment]
+1

**Pull request merged**